.changeset/metal-snakes-wave.md

```diff
@@ -0,0 +1,19 @@
+---
+"@rhds/elements": minor
+---
+
+✨ Added `<rh-breadcrumb>`.
+
+A breadcrumb navigation is a secondary navigation element consisting of a list
+of links to the parent pages of the current page in hierarchical order. It
+helps users find their place within a website or web application.
+
+```html
+<rh-breadcrumb>
+  <ol>
```

Comment:
have you considered the potential advantages of using internal roles and custom children, like this hypothetical example:

```html
<rh-breadcrumbs>
  <rh-breadcrumb href="../../../..">Home</rh-breadcrumb>
  <rh-breadcrumb href="../../../">Home</rh-breadcrumb>
  <rh-breadcrumb href="#" current>Our Team</rh-breadcrumb>
</rh-breadcrumb>
```

Tell me a bit of your thought process for preferring nested ol/li with aria?

Reply:
I went with the current implementation because:

  * It's straightforward, produces few [WTF's per minute](https://github.com/ryanmcdermott/clean-code-javascript?tab=readme-ov-file#introduction) at first glance.
  * Works without JS
  * Works with DSD
  * Is semantically correct & recommended in the [Breadcrumbs](https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/) WAI ARIA Authoring Practices Guide.
  * Easy to implement by CMS authors who possibly don't have access to output a custom menu (or don't want to)

Trade offs:

  * Relies on lightdom styles / no style encapsulation
  * Other trade offs...?

If there are good reasons for making this `<rh-breadcrumbs>` & nested `<rh-breadcrumb>` tags, I'm open to that discussion and the resulting changes.